### PR TITLE
My Portfolio: Work-about section

### PR DESCRIPTION
**Pull request by PromzzyKoncepts**
Hey there,
I trust you are doing great
Here are the changes I made in this repository
- Added the Work and about-me sections of the portfolio
- Committed the changes
- Edited my Read and attached a Live Demo link

Please leave a good rating if you love the portfolio
Thanks a lot!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+              <div class="work-section-ul">
+                <ul>
+                  <li class="btn">css</li>
+                  <li class="btn">html</li>
+                  <li class="btn">bootstrap</li>
+                  <li class="btn">Ruby</li>
+                </ul>
+              </div>
+              <button type="button" id="project-button1" class="project-btn1 project-btn">See Project</button>
+            </section>
```
Comment (vic778) [inline index.html:91]: - [ ] Kindly the font size , the design and the text-color are not match to the figma template as requested in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 
Your design
Screenshot from 2022-06-30 22-54-31
Figma design
Screenshot from 2022-06-30 22-53-57

File: index.html [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+          </div>
```
Comment (vic778) [inline index.html:92]: - [ ] Kindly delete this `div` because  it's not closing anything else

File: index.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+      </section>
+
+      <section id="About" class="about-me">
+        <h2 class="about-me-text">About me</h2>
+        <p class="about-me-paragraph">Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.
+        </p>
+        <button type="button" class="project-btn1 about-btn">Get my Resume</button>
```
Comment (vic778) [inline index.html:227]: - [ ] Kindly change the color of the button also.

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @PromzzyKoncepts 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (PromzzyKoncepts) [reply index.html:91]: @vic778 
Hello,
Thanks for the review
Although I think you got this one mixed up
I am using template 2 but you seem to be comparing it with template 1
Kindly recheck template 2

If am wrong, please respond.

File: index.html (hunk 3 above)
Comment (PromzzyKoncepts) [reply index.html:227]: okay done
Thanks!
@vic778

**Comment by PromzzyKoncepts** [comment]
@vic778 
Thanks for the review, however i am confused on some checks you made.
kindly check them out and reply

File: index.html (hunk 1 above)
Comment (vic778) [reply index.html:91]: @PromzzyKoncepts Kindly my portfolio also is template 2. the color of the see-button first is not match to the Figma template as you can see at this screenshot . secondly the font-family you're using is not match. you may used the `font-family: Inter, sans-serif;` to make your text looks like the figma

**Comment by vic778** [comment]
@PromzzyKoncepts  what i saw telling you is when you launch the app basically it should be lunched like this 
Figma template 
Screenshot from 2022-07-01 08-03-29
## there,s no space around the pages. but for you when the app is lunched you can see like it's in a frame.  kindly inspect the page and you will see like it has a padding everywhere in the top right left.. see the  screenshot i made when i inspected the page.  
Your design
Screenshot from 2022-06-30 22-40-33

**Comment by PromzzyKoncepts** [comment]
Hi there!
i have successfully made corrections to the  HTML and CSS of the project
Pls kindly drop a review as soon as you can.

**Review by algerina — APPROVED** [review]
Hi @PromzzyKoncepts , 
Good job so far!

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:


Great work on implementing the changes requested by the previous TSEs
 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**